shared/src/price_estimation/gas.rs

```diff
@@ -22,5 +25,13 @@ pub const TRADE: u64 =
 /// of the 90% most traded tokens by volume in the month of Oct. 2021.
 pub const ERC20_TRANSFER: u64 = 27_513;
 
+/// lower bound for gas refunds
+/// this number was derived from some empiric observations
+pub const GAS_REFUNDS: u64 = 13_000;
+
 /// a settlement that contains one trade
-pub const SETTLEMENT_SINGLE_TRADE: u64 = SETTLEMENT + TRADE + 2 * ERC20_TRANSFER;
+pub const SETTLEMENT_SINGLE_TRADE: u64 =
```

Comment:
nanonit: Since this is only used in a single place now, maybe it should be defined there?

Reply:
Personally, I'm slightly partial to having all our gas constants in one place.